### ISSUE TITLE
Is there a way i can provide properties in teaxtArea like only 200 character are allowed 

### ISSUE THREAD

**Issue by nirmesh** [rekit/antd-form-builder#112]
Is there a way i can provide properties in teaxtArea like only 200 character are allowed and shows the count for remaining words etc while user is writing

**Comment by JulesLecuir** [comment]
Yes, you can pass a `rules` argument to your field, like you would for a regular Antd component. So you can ask for `{max: 200}` in the rules, it shoudl work (I haven't tested though). The Antd docs should help https://ant.design/components/form#formitem
